tests/Unit/Domain/Test_Neighbors.cpp

```diff
@@ -0,0 +1,42 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Domain/Neighbors.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+TEST_CASE("Unit.Domain.Neighbors", "[Domain][Unit]") {
+  // Test default constructor:
+  Neighbors<3> test_neighbors;
+  static_cast<void>(test_neighbors);
+
+  // Test constructor:
+  const Orientation<3> custom_orientation(std::array<Direction<3>, 3>{
+      {Direction<3>::upper_eta(), Direction<3>::upper_zeta(),
+       Direction<3>::upper_xi()}});
+
+  std::unordered_set<ElementId<3>> custom_ids;
+  auto segment1_ids = std::array<SegmentId, 3>(
```

Comment:
`std::array<...> segment1_ids{{...}};` seems clearer.  (Also for the following cases.)